iot_backend_py/settings.py

```diff
@@ -124,3 +126,31 @@
 # https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+LOGGING = {
```

Comment:
Will check logging structure later

Reply:
Ok, for now, can it be included? or should I remove the logging completely?

Reply:
let's keep it